package com.josecjuniors.logossrv.core.progression.application.port.in;

import com.josecjuniors.logossrv.core.progression.application.service.ProgressionOutcome;
import com.josecjuniors.logossrv.core.progression.domain.model.SubjectId;
import com.josecjuniors.logossrv.core.registroatividade.application.service.ProgressionInput;

public interface ExecuteSubjectProgressionUseCase {

    ProgressionOutcome execute(SubjectId subjectId, ProgressionInput input);
}
